docs/started/releasepolicy.md

```diff
@@ -81,3 +81,53 @@ server.
 | (0.3)           |                                           | 2015-03        | 2015-07              |
 | (0.2)           |                                           | 2014-10        | 2015-03              |
 | (0.1.7)         |                                           | 2014-01        | 2014-10              |
+
+## Secondary Rucio Software Policy
+
+### WebUI
+
+The Rucio WebUI is an initiative to modernize the Rucio user interface by
+leveraging the latest web technologies. Built with Next.js, TailwindCSS, and
+React.js, this new WebUI aims to provide a robust architecture designed to
+serve the community for the next decade, much like its predecessor.
+
+#### Key Technologies
+
+- **Next.js**: Provides a flexible and efficient framework for server-side
+  rendering and static site generation.
+- **TailwindCSS**: Enables rapid and responsive UI development with
+  utility-first CSS.
+- **React.js**: Facilitates dynamic and interactive user interfaces with a
+  component-based approach.
+
+#### Versioning
+
+The versioning of the Rucio WebUI is closely aligned with the Rucio Server's
+release versioning. Both follow semantic versioning principles, ensuring
+consistency and compatibility.
+
+- **Major Version**: The major version number of the WebUI matches that of the
+  Rucio Server. For example, Rucio Server version 35.x.x is compatible with
+  Rucio WebUI version 35.x.x.
+- **Minor and Patch Versions**: These versions follow semantic versioning:
+   - **Minor Version**: Incremented for new features and improvements that are
+     backward-compatible.
+   - **Patch Version**: Incremented for backward-compatible bug fixes.
+
+### JupyterLab extension
+
+The Rucio JupyterLab extension does not follow the relase policy described
```

Comment:
Change to: The Rucio Jupyterlab extension follows it's own release policy described below.